### PR TITLE
Remove obsolete file sharing information

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

I removed obsolete information related to file sharing on Windows from the Docker Desktop docs.
Since we released Docker Desktop Stable 2.2.0.x and Edge 2.2.1.0 the new file sharing implementation does not need credentials or firewall configuration anymore.

### CODE REVIEW

File: docker-for-windows/index.md [hunk 1]
```diff
@@ -248,30 +243,11 @@ username and password. Enter your credentials and then click **Apply & Restart**
...
-To share the drive, allow connections between the Windows host machine and the
-virtual machine in Windows Firewall or your third party firewall software. You
-do not need to open port 445 on any other network.
-
-By default, allow connections to `10.0.75.1` on port 445 (the Windows host) from
-`10.0.75.2` (the virtual machine). If your firewall rules seem correct, you may
-need to toggle or
-[reinstall the File and Print sharing service on the Hyper-V virtual network card](http://stackoverflow.com/questions/42203488/settings-to-windows-firewall-to-allow-docker-for-windows-to-share-drive/43904051#43904051)
-
 #### Shared drives on demand
```
Comment (thaJeztah) [inline docker-for-windows/index.md]: The changes don't affect this? (as in: this feature is still "current"?)

File: docker-for-windows/index.md [hunk 2]
```diff
@@ -248,30 +243,11 @@ username and password. Enter your credentials and then click **Apply & Restart**
 >
  * Docker Desktop sets permissions to read/write/execute for users, groups and others [0777 or a+rwx](http://permissions-calculator.org/decode/0777/).
    This is not configurable. See [Permissions errors on data directories for shared volumes](troubleshoot.md#permissions-errors-on-data-directories-for-shared-volumes).
->
- * Ensure the domain user has access to shared drives, as described in [Verify domain user has permissions for shared drives](troubleshoot.md#verify-domain-user-has-permissions-for-shared-drives-volumes).
```
Comment (thaJeztah) [inline docker-for-windows/index.md]: Do we know what share / number of users are still on older versions? Thinking if we need to move the content to a separate page, and add a mention "if you're using version XX or older", which either instructs them to upgrade to the latest & greatest, or point to the troubleshooting page for older versions.


/cc @usha-mandya @stephen-turner @nebuk89  for thoughts

File: docker-for-windows/index.md (hunk 1 above)
Comment (mat007) [reply docker-for-windows/index.md]: It’s a separate feature, still «current».

File: docker-for-windows/index.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/index.md]: I believe we support our latest and greatest on Docker Desktop. Therefore, not sure if moving obsolete features to another page and duplicating the entire topic is a good option.  Users can browse through the release notes to identify features added/removed from specific versions.

Comment (stephen-turner) [reply docker-for-windows/index.md]: I agree with Usha. We can't keep documenting all old versions when we change something.

Comment (thaJeztah) [reply docker-for-windows/index.md]: Okay, thanks for checking. I've seen this error/situation popping up in many situations (some users linger a bit behind "latest & greatest"), so wanted to double check 👍

**Comment by usha-mandya** [comment]
Thank you @thaJeztah.

**Pull request merged**